app/assets/javascripts/thanks.js

```diff
@@ -0,0 +1,30 @@
+document.addEventListener('DOMContentLoaded', function () {
+
+    const allCookies = document.cookie;
+    console.log(`All Cookies: ${allCookies}`);
+
+    const cookiesArray = allCookies.split(';');
+
+    let locale = 'en';
+
+    for (const cookie of cookiesArray) {
```

Comment:
Can we do this without using loop and directly fetching cookie with name = 'locale'